--- squish-1.10/Makefile.orig	2006-04-07 19:30:11.000000000 +0200
+++ squish-1.10/Makefile	2010-10-16 07:53:52.799740205 +0200
@@ -3,26 +3,25 @@
 
 SRC = alpha.cpp clusterfit.cpp colourblock.cpp colourfit.cpp colourset.cpp maths.cpp rangefit.cpp singlecolourfit.cpp squish.cpp
 
-OBJ = $(SRC:%.cpp=%.o)
+OBJ = $(SRC:%.cpp=%.lo)
 
-LIB = libsquish.a
+LIB = libsquish.la
 
 all : $(LIB)
 
 install : $(LIB)
-	install squish.h $(INSTALL_DIR)/include 
-	install libsquish.a $(INSTALL_DIR)/lib
+	install squish.h $(DESTDIR)$(includedir)
+	libtool --mode=install install libsquish.la $(DESTDIR)$(libdir)
 
 uninstall:
 	$(RM) $(INSTALL_DIR)/include/squish.h
 	$(RM) $(INSTALL_DIR)/lib/libsquish.a
 
 $(LIB) : $(OBJ)
-	$(AR) cr $@ $?
-	ranlib $@
+	libtool --mode=link --tag=CXX $(CXX) -o $@ $(CXXFLAGS) $(LDFLAGS) -rpath $(libdir) $<
 
-%.o : %.cpp
-	$(CXX) $(CPPFLAGS) -I. $(CXXFLAGS) -o$@ -c $<
+%.lo : %.cpp
+	libtool --mode=compile --tag=CXX $(CXX) $(CPPFLAGS) -I. $(CXXFLAGS) -o$@ -c $<
 
 clean :
 	$(RM) $(OBJ) $(LIB)
