--- squish-1.10/alpha.cpp.orig	2006-06-29 14:43:24.000000000 +0200
+++ squish-1.10/alpha.cpp	2010-10-16 08:31:13.411734339 +0200
@@ -25,6 +25,7 @@
    
 #include "alpha.h"
 #include <algorithm>
+#include <climits>
 
 namespace squish {
 
--- squish-1.10/singlecolourfit.cpp.orig	2007-03-21 20:43:59.000000000 +0100
+++ squish-1.10/singlecolourfit.cpp	2010-10-16 08:39:11.391740205 +0200
@@ -27,6 +27,8 @@
 #include "colourset.h"
 #include "colourblock.h"
 
+#include <climits>
+
 namespace squish {
 
 struct SourceBlock
